x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/AllocationRoutedStep.java

```diff
@@ -70,8 +70,8 @@ public Result isConditionMet(Index index, ClusterState clusterState) {
     static int getPendingAllocations(Index index, AllocationDeciders allocationDeciders, ClusterState clusterState) {
         // All the allocation attributes are already set so just need to check
         // if the allocation has happened
-        RoutingAllocation allocation = new RoutingAllocation(allocationDeciders, clusterState.getRoutingNodes(), clusterState, null,
-            System.nanoTime());
+        RoutingAllocation allocation = new RoutingAllocation(ALLOCATION_DECIDERS, clusterState.getRoutingNodes(), clusterState, null,
```

Comment:
This change seems odd to me. I suppose the `allocationDeciders` are effectively unused, but in that case I would still prefer to pass down `allocationDeciders`.

Reply:
Thanks for catching this! It seems that git's merge algorithm did not report a conflict when I merged master back to this branch :(